pkg/volume/csi/csi_mounter.go

```diff
@@ -238,14 +240,15 @@ func (c *csiMountMgr) SetUpAt(dir string, mounterArgs volume.MounterArgs) error
 	if err != nil {
 		return volumetypes.NewTransientOperationFailure(log("mounter.SetUpAt failed to assemble volume attributes: %v", err))
 	}
-	if podAttrs != nil {
-		if volAttribs == nil {
-			volAttribs = podAttrs
-		} else {
-			for k, v := range podAttrs {
-				volAttribs[k] = v
-			}
+	volAttribs = mergeMap(volAttribs, podAttrs)
```

Comment:
A side effect that we need to think about is since this function will be called much more frequently for the plugins that enable it, are there any log statements we need to conditionally not log?

Reply:
looking at the SetupAt function which will be called more frequently: all informational logs are V(4) and other logs are all about errors.

Reply:
I need to double check the log level for other messages in the remount path.

Reply:
Looks like we use log level 5 for logs during remount: https://github.com/kubernetes/kubernetes/blob/e23d83eead3b5ae57731afb0209f4a2aaa4009dd/pkg/kubelet/volumemanager/reconciler/reconciler.go#L271